src/winetricks

```diff
@@ -11971,6 +11971,60 @@ __EOF__
 
 #----------------------------------------------------------------
 
+w_metadata office2013pro apps \
+    title="Microsoft Office 2013 Professional" \
+    publisher="Microsoft" \
+    year="2013" \
+    media="cd" \
+    file1="setup.exe" \
+    installed_file1="$W_PROGRAMS_X86_WIN/Microsoft Office/Office15/WINWORD.EXE"
+
+load_office2013pro()
+{
+    if [ "$W_ARCH" = "win64" ]; then
+        w_die "Installation of Office 2013 Pro will not succeed on a 64bit wine prefix"
+    fi
+
+    if [ ! -x "$(which winbindd 2>/dev/null)" ]; then
+        w_warn "winbind is required"
```

Comment:
If it's required, it should be w_die.

Aside from that, I'm pretty sure you meant ntlm_auth, not winbindd. At least on debian, winbindd is in sbin, whereas ntlm_auth is in /usr/bin. AFAIK wine doesn't use winbindd (the only references to it are in comments.

Reply:
`ntlm_auth` and `winbindd` are both installed together with winbind so I was actually copying the part from [Adobe Digital Editions](https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L11043). Will change that to `ntlm_auth`.